### PR TITLE
Make highlight suspect/banned words regex case insensitive

### PR DESCRIPTION
**Pull request by ALawliet**
## What does this PR do?
Fixes: https://www.pivotaltracker.com/story/show/147775935

## How do I test this PR?
- Configure > Moderation Settings > Add to Suspect words list "fishy" and "amenabar"
- create (non-admin) comments with "Fishy", "Amenabar", "fishy", "amenabar"
- notice they are now all highlighted regardless of casing

screen shot 2017-06-24 at 3 44 12 pm

### CODE REVIEW

**Comment by kgardnr** [comment]
This LGTM! Once @bel or @cvle sign off on code review, I'm good to merge. Thanks @ALawliet!

**Review by alinhle — APPROVED** [review]

**Pull request merged**
